backend/db/migrate/20230404161002_exports_cutoff_at.rb

```diff
@@ -1,5 +1,14 @@
 class ExportsCutoffAt < ActiveRecord::Migration[6.1]
-  def change
+  def up
+    remove_column :studies, :shareable_after_months, :integer
+    add_column :studies, :public_on, :datetime, null: true
     add_column :analysis_response_exports, :cutoff_at, :datetime
+
+  end
+
+  def down
+    remove_column :studies, :public_on, :datetime
+    add_column :studies, :shareable_after_months, :integer
+    remove_column :analysis_response_exports, :cutoff_at, :datetime
   end
```

Comment:
no reason to do `up` & `down` migrations, add_column and remove_column are both reversible and only require `change`

Reply:
got it - thanks! copy/paste syndrome got me